android/app/src/main/java/com/woowacourse/ody/presentation/meetings/MeetingsViewModel.kt

```diff
@@ -47,18 +47,8 @@ class MeetingsViewModel(
                 }
         }
 
-    override fun navigateToEtaDashboard(meetingId: Long) {
-        viewModelScope.launch {
-            meetingRepository.fetchMeeting(meetingId).onSuccess {
-                _navigateAction.postValue(
-                    MeetingsNavigateAction.NavigateToEtaDashboard(meetingId = it.id),
-                )
-            }.onFailure {
-                analyticsHelper.logNetworkErrorEvent(TAG, it.message)
-                Timber.e(it)
-            }
-        }
-    }
+    override fun navigateToEtaDashboard(meetingId: Long) =
+        _navigateAction.postValue(MeetingsNavigateAction.NavigateToEtaDashboard(meetingId))
```

Comment:
```suggestion
    override fun navigateToEtaDashboard(meetingId: Long) {
        return _navigateAction.postValue(MeetingsNavigateAction.NavigateToEtaDashboard(meetingId))
    }
```
위와 같은 방식은 어떤가요? IDE 상 한줄에 다 드러나지 않는 경우에는 `=` 대신 `{ return ~~ }`을 사용하기로 정했던 것 같아요. 컨벤션 문서 찾아보니까 따로 적혀있진 않네요.. ;;; 제가 잘못 기억하는 걸 수도

Reply:
컨벤션에 적어두고 수정하겠습니다잉

Reply:
어떤 방식이든 타입 명시하면 좋을거 같아요!
만약 Unit 타입이면 중괄호 써주기 어떤가요?

Reply:
아 줄바꿈까지만 컨벤션이군요~! 저는 다 좋습니다. 정말 취향 차이인 것 같네요.

Reply:
줄바꿈 후 중괄호로 가겠습니다!